### ISSUE TITLE
Optional improvement: Adding scores of the favourite dog breed on the selected traits

### ISSUE THREAD

**Issue by shivajena** [UBC-MDS/doggodash#52]

**Comment by stevenleung2018** [comment]
@shivajena Do you mean adding to the tooltip of the "Your Top 5 Dog Breeds" plot (top right)?

**Comment by shivajena** [comment]
@stevenleung2018 not exactly. There could be a white output box, which takes input from the sliders and outputs the scores the breed for those specific traits

**Comment by stevenleung2018** [comment]
Thanks for the clarification.  Since there is already code for calculating the scores, I may not be able to contribute much to implement this feature.  Sorry.  Let's see who else can take this up.

**Comment by shivajena** [comment]
Yes, lets park this for future consideration
